src/pages/Members/Members.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './styles.scss';
+
+import memberGroupIcon from '@/assets/icons/member-group.svg';
+import MemberButtons from '@/components/TopBar/components/MemberButtons/MemberButtons';
+import { useWorkspace } from '@/hooks/useWorkspace';
+
+const Members = () => {
+  const { members, activeWorkspace } = useWorkspace();
+
+  const renderHeader = () => {
+    return (
+      <div className="members-header">
+        <h2>Members</h2>
+        <div className="members-header-paragraph">
+          <img alt="member icon" src={memberGroupIcon} />
+          <span>Members area</span> is dedicated to managing the members associated to this
+          workspace.
+        </div>
+      </div>
+    );
+  };
+
+  const renderButtons = () => {
+    return <MemberButtons />;
+  };
+
+  const constructTableHeader = (name: string) => {
+    return <div className="members-table-header">{name}</div>;
+  };
+
+  const renderTable = () => {
```

Comment:
So there's a few things missing still:
- I can see in the design that there is pagination which we'd need to include.
- Icons next to the table headings
- The bin icon next to the user, which I assume would replace the `remove member` button 
- There should be all of the columns, even if the data for it isn't there yet
- The rows seem to be not aligned centrally.